leipzig-gloss-examples.typ

```diff
@@ -66,73 +86,256 @@ See https://www.eva.mpg.de/lingua/pdf/Glossing-Rules.pdf
+    morphemes: ([1#pl], [#com], [Marko], [go-#pst\-#pl], [bus-#ins], [#all], [Peredelkino]),
+    gloss_lines: (
+        ([we], [with], [Marko], [go-#pst\-#pl], [bus-by], [to], [Peredelkino]),
+        ([nous], [avec], [Marco], [allés], [en bus], [à], [Peredelkino]),
+        ([私たちは],[と], [マルコ], [行きました], [バスで], [に], [ペレデルキノ])
+    ),
+    line_styles: (
+        none,
+        text.with(style: "italic", fill: purple, font: "Noto Serif"),
+        text.with(font: "Noto Sans CJK JP", fill: red),
+    ),
+    translation: "Marko and I went to Peredelkino by bus",
+    translation_style: text.with(style: "italic", weight: "bold")
+)
+```
+])
+)
 )
 
-#numbered_gloss(
+== Spacing \& padding customization
+
+This package allows you to customize the spacing used for various elements in the gloss. The relevant spacing parameters are as follows:
+
+/ `interword_spacing`: (Default: 1em) the amount of horizontal space between each word/item in the gloss.
+/ `gloss_line_spacing`: (Default: 0.5em) the amount of vertical space between lines of the gloss.
+/ `post_header_space`: (Default: 0.5em) the amount of extra vertical spacing added between the header text and the gloss itself. If set to `none`, a `linebreak()` will be used.
+/ `pre_translation_space`: (Default: 0.5em) the amount of extra vertical spacing added between the gloss itself and the translation text. If set to `none`, a `linebreak()` will be used.
+/ `gloss_padding`: (Default: 2em) the size of the padding from the left margin for the entire gloss example. This length marks where the text itself starts for both numbered and un-numbered glosses, allowing for consistent alignment between the two.
+/ `left_padding`: (Default: .5em) the size of the padding from the left margin for the number of a numbered gloss. Should be less than `gloss_padding` to avoid ugly overlaps. Is completely ignored by un-numbered glosses.
+
+While these parameters are set to sensible defaults, feel free to customize them as needed for your use case.
+
+#block(
+    breakable: false,
+stack(
+    dir:ttb,
+    spacing: 1em,
+    [#numbered_gloss(
     header_text: [Hittite (Lehmann 1982:211)],
     source_text: ([n=an], [apedani], [mehuni],[essandu.]),
     morphemes: ([#smallcaps[conn]=him], [that.#dat.#sg], [time.#dat.#sg], [eat.they.shall]),
     translation: "They shall celebrate him on that date",
+    gloss_line_spacing: 1em,
+    interword_spacing: 3em,
+    post_header_space: 2em,
+    pre_translation_space: none,
+    gloss_padding: 4em,
+    left_padding: 2em,
+)],
+    block(
+  fill: luma(230),
+  inset: 8pt,
+  radius: 4pt,
+[```typst
+    #numbered_gloss(
+        header_text: [Hittite (Lehmann 1982:211)],
+        source_text: ([n=an], [apedani], [mehuni],[essandu.]),
+        morphemes: ([#smallcaps[conn]=him], [that.#dat.#sg], [time.#dat.#sg], [eat.they.shall]),
+        translation: "They shall celebrate him on that date",
+    )
+    ```
```

Comment:
This code block needs to be updated with the actual custom values for spacing that you used.